etc/picongpu/juwels-jsc/booster_picongpu.profile.example

```diff
@@ -63,7 +63,7 @@ export LD_LIBRARY_PATH=$PNGwriter_DIR/lib:$LD_LIBRARY_PATH
 export LD_LIBRARY_PATH=$ADIOS2_ROOT/lib64:$LD_LIBRARY_PATH
 export LD_LIBRARY_PATH=$OPENPMD_ROOT/lib64:$LD_LIBRARY_PATH
 
-export PATH=$ADIOS_ROOT/bin:$PATH
+export PATH=$ADIOS2_ROOT/bin:$PATH
```

Comment:
Note: this was probably a bug in the profile? It was alrady using ADIOS2, but this variable was still set for ADIOS1